web/src/settings_panel_menu.js

```diff
@@ -135,7 +135,7 @@ export class SettingsPanelMenu {
         $(".settings-section").removeClass("show");
 
         settings_sections.load_settings_section(section);
-
+        
```

Comment:
I'm surprised the linter didn't catch this, but watch out for whitespace like this and be sure to remove it.